packages/components/button/src/context.ts

```diff
@@ -0,0 +1,4 @@
+import type { InjectionKey } from 'vue'
+import type { ElButtonGruopContext } from './button.types'
+
+export const ElButtonGroupKey: InjectionKey<ElButtonGruopContext> = Symbol()
```

Comment:
```diff
--- ElButtonGroupKey
+++ elButtonGroupKey
```
I think the first letter lowercase would be better, so that you can match the style of the code.

Reply:
By the way, rename `context.ts` to `token.ts` may be more adapted to the existing filename.

Reply:
Changed. Thank you.